go.mod

```diff
@@ -0,0 +1,3 @@
+module boringssl.googlesource.com/boringssl
```

Comment:
Is this needed?

Reply:
yes, there are dependencies on the upstream Go source for reading the symbol names. if I remove this, we get:
```
go run util/read_symbols.go -out symbols.txt target/debug/build/ring-25e4c3b394b2836b/out/libring-core.a
build github.com/ramosbugs/ring/util: cannot find module for path boringssl.googlesource.com/boringssl/util/ar
```